Static/Row.swift

```diff
@@ -108,6 +108,12 @@ public struct Row: Hashable, Equatable {
     /// View to be used for the row.
     public var cellClass: CellType.Type
 
+    /// Nib to be used for the row
+    public var cellNib: UINib?
+    
+    // Row height
```

Comment:
Looks like this is missing a `/`. Can you add that in so the property has documentation, albeit sparse? :smile: